example.config.json

```diff
@@ -2,15 +2,12 @@
   "githubId": "GITHUB_USER_ID",
   "user": {
     "botname": "GITHUB_USER_ID",
-    "appHost": "http://localhost:7000",
-    "apiServer": "www.freecodecamp.com",
-    "appRedirectUrl": "http://localhost:7891/login/callback"
+    "apiServer": "freecodecamp.com"
   },
   "rooms": [
     {
       "title": "bothelp",
       "name": "GITHUB_USER_ID/test",
-      "icon": "question",
```

Comment:
These values (icon, appHost, and appRediectUrl) are used in other places where built-in bot configs are provided and should be removed from there as well.